### ISSUE TITLE
Avoid italic text for better accessibility and readability

### ISSUE THREAD

**Issue by abuyoyo** [CMB2/CMB2#1404]
In line with recent WordPress core style-guide/accessibility guidelines, italic text should be avoided where possible.
Perhaps cmb2-description and labels should not be in italics either.

Reference to wp-core ticket/task: https://core.trac.wordpress.org/ticket/47327

**Comment by jtsternberg** [comment]
Good call. Thank you for the info!
